plugin/src/test/java/testcode/spring/SpringEntityLeakController.java

```diff
@@ -0,0 +1,24 @@
+package testcode.spring;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class SpringEntityLeakController {
+	@RequestMapping("/api1")
+	public SampleEntity api1(@RequestParam("url") String url) {
+		return new SampleEntity("entity1");
+	}
+
+	@RequestMapping("/api2")
+	public SampleEntityTwo api2(@RequestParam("url") String url) {
```

Comment:
It would be nice to also cover List of Entity.
I know type erasure is a thing but it could be infer based on casting done inside the method.